### PR TITLE
MORS modal content

### PR DESCRIPTION
**Pull request by gillycheesesteak**

### CODE REVIEW

File: src/components/modal/content/US-EZP/parts/EZP.jsx [hunk 1]
```diff
@@ -8,19 +8,12 @@ import Icon from '../../../parts/Icon';
...
-    ['truck', "If your items ship separately, we'll split each new shipment into equal monthly payments."],
-    ['tags', 'Your PayPal Credit minimum monthly payment includes all Easy Payments and other PayPal Credit purchases.']
-];
-
 export const Header = () => {
     const buttonRef = useRef();
     const handleApplyNowClick = useApplyNow('Apply Now');
+    const { products } = useServerData();
+
+    const { content } = products.find(product => product.meta.product === 'EZP');
```
Comment (paypal-are) [inline src/components/modal/content/US-EZP/parts/EZP.jsx]: Does `find` work in IE?

File: src/components/modal/content/DE/parts/Disclaimer.jsx [hunk 2]
```diff
@@ -9,17 +10,14 @@ const Disclaimer = ({ terms }) => {
         terms.offers &&
         terms.offers.length > 0
     ) {
-        const minAmount = terms.formattedMinAmount.split(',')[0];
-        const maxAmount = terms.formattedMaxAmount.split(',')[0];
+        const content = useContent('INST');
```
Comment (Seavenly) [inline src/components/modal/content/DE/parts/Disclaimer.jsx]: Can this be lifted out of the `if` block and used for both logic branches?

File: src/components/modal/content/DE/parts/TermsTable.jsx [hunk 3]
```diff
@@ -1,45 +1,34 @@
...
-        );
+        return <h3 className="error">{content.terms.maxAmount}</h3>;
     }
 
     const [offer] = terms.offers.length ? terms.offers : [];
     if (!offer || !offer.qualified) {
         return genericError;
     }
 
+    const offerContent = useContent('INST');
```
Comment (Seavenly) [inline src/components/modal/content/DE/parts/TermsTable.jsx]: Why are we calling `useContent('INST')` twice? Can the above instance not be used here?

File: src/components/modal/content/GB/parts/PL.jsx [hunk 4]
```diff
@@ -46,15 +55,18 @@ const PL = () => {
                 <div className="info">
                     <Icon name="shopping-bag" />
                     <p>
-                        Get your items straight away <br /> and pay nothing for 1 month.
+                        {content.instructions[0][0]} <br />
```
Comment (Seavenly) [inline src/components/modal/content/GB/parts/PL.jsx]: Should the icon name be included with the instruction content and them just loop over them?

File: src/components/modal/content/DE/parts/INST.jsx [hunk 5]
```diff
@@ -3,34 +3,39 @@ import { h } from 'preact';
...
-    </section>
-);
+    return (
+        <section className="content-body">
+            <Carousel
+                items={[
+                    {
+                        imageSrc: 'https://www.paypalobjects.com/upstream/assets/img/de/icon_shoppingcart.svg',
+                        imageAlt: 'Shopping cart icon',
+                        description: content.carousel[0]
```
Comment (Seavenly) [inline src/components/modal/content/DE/parts/INST.jsx]: `imageSrc` and `imageAlt` should possibly be moved into the content? Then we can just loop over `content.carousel`

File: src/components/modal/content/US-EZP/parts/EZP.jsx [hunk 6]
```diff
@@ -56,6 +47,11 @@ export const Content = () => {
     const { onClick } = useXProps();
     const { aprEntry } = useServerData();
 
+    const content = useContent('EZP', {
+        aprEntry,
+        fullYear: new Date().getFullYear()
```
Comment (Seavenly) [inline src/components/modal/content/US-EZP/parts/EZP.jsx]: Does this do anything now?

File: src/components/modal/content/US/parts/PI4.jsx [hunk 7]
```diff
@@ -1,30 +1,23 @@
...
-const instructions = [
-    'Pay in 4 is not available to residents of Wisconsin, North Dakota, Missouri, or any U.S. Territories.',
-    'Late fees may apply for missed payments depending on your state of residency',
-    'When applying, a soft credit check may be needed, but will not affect your credit score.'
-];
-
 export default () => {
     const { terms } = useServerData();
 
+    const content = useContent('PI4');
```
Comment (Seavenly) [inline src/components/modal/content/US/parts/PI4.jsx]: Should we consider using a more generic product name for the future?

**Pull request merged**